src/Chart.jsx

```diff
@@ -330,17 +325,14 @@ class Chart extends Component {
         const bisect = d3.bisector(d => d[0]).left;
         const idx = bisect(d.data, x0);
         const y = d.data[idx] && d.data[idx][1];
+        let labelText = '';
+        let x1 = x0 && d3.format(xLabelFormat)(x0);
+        let y1 = y && y.toFixed(2);
 
-        let labelText = tooltipLabel;
-        if (d.tooltipLabelFn && x0 && y) {
-          labelText = d.tooltipLabelFn(
-            x0 && d3.format(xLabelFormat)(x0),
-            y.toFixed(2)
-          );
-        }
-        d3.select(this).select('text').text(labelText);
+        if (x1 && y1) labelText = y1 + 'years ,for ' + x1 + 'interest rate';
 
-        return 'translate(' + margin.left + ',' + i * 15 + ')';
```

Comment:
This is for `transform` attribute. If you don't intend to use `transform`, then probably use `selection.each`. Ex, right now, `transform` is getting `undefined` value.